docs/sources/installation/helm/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Install Grafana Loki with Helm
+menuTitle: Deploy on Kubernetes
+description: Use this template when you write a concept topic.
```

Comment:
@jeschkies, please add a description in a subsequent PR to remove this placeholder text.